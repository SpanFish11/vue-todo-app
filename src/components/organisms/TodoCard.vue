<script setup>
import TodoList from '@/components/molecules/TodoList.vue'
import TodoActions from '@/components/molecules/TodoActions.vue'
</script>

<template>
  <div class="card shadow-lg">
    <TodoList />
    <TodoActions />
  </div>
</template>
