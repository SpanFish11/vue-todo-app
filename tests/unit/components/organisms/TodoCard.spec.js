import { beforeEach, describe, expect, it } from '@jest/globals'
import { shallowMount } from '@vue/test-utils'
import TodoCard from '@/components/organisms/TodoCard.vue'

describe('Given the TodoCard component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(TodoCard, {})
  })

  it('should render', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
