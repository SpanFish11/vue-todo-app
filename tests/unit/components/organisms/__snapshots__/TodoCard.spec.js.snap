// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the TodoCard component should render 1`] = `
<div
  class="card shadow-lg"
>
  <todo-list-stub />
  <todo-actions-stub />
</div>
`;
